cms/templates/js/xblock-access-editor.underscore

```diff
@@ -0,0 +1,5 @@
+<div class="incontext-editor-action-wrapper">
+    <a href="" class="unit-container visibility-button" title="<%- gettext('Access Settings') %>">
```

Comment:
Good question! I also spent some time searching for this. I'm pretty sure that it is mentioned in the Edge a11y course, but unfortunately our courses aren't searchable. I read though some comments in this stack overflow thread-- https://stackoverflow.com/questions/5637969/is-an-empty-href-valid.

Here is a good article by an accessibility expert on this topic: https://marcysutton.com/links-vs-buttons-in-modern-web-applications/.

FYI @cptvitamin